### ISSUE TITLE
View exceptions hide stack trace for the the actual exception

### ISSUE THREAD

**Issue by jasonvarga** [facade/ignition#107]
Using this silly example:

``` php
class Thing
{
    function url()
    {
        return route('unknown');
    }
}

```
``` php
Route::get('/', function () {
    return view('welcome', ['thing' => new \App\Thing]);
});
```
```
{{ $thing->url() }}
```
Without Ignition, you can see the trace from the view through to the exception.
With Ignition, it goes straight from the PhpEngine to the final exception.

![image](https://user-images.githubusercontent.com/105211/64381384-0f322500-d001-11e9-8453-93c12ce155a9.png)

![image](https://user-images.githubusercontent.com/105211/64381469-46083b00-d001-11e9-882e-8455b720bd14.png)

**Comment by jasonvarga** [comment]
I'm sure you would know better than me, but maybe this could be a clue.

If I change the first argument inside `Stacktrace::createForThrowable` to the previous exception, it displays the correct (?) trace.

https://github.com/facade/flare-client-php/blob/master/src/Stacktrace/Stacktrace.php#L14

```diff
return new static(
-    $throwable->getTrace(), 
+    $throwable->getPrevious()->getTrace(), 
    $throwable->getFile(), $throwable->getLine()
);
```

![image](https://user-images.githubusercontent.com/105211/64382326-6e913480-d003-11e9-8635-d4ad1cf62f30.png)

**Comment by mpociot** [comment]
Nice find @jasonvarga 

Yeah when we catch those view exceptions, we throw a new custom exception that holds all that extra view information. This custom ViewException then gets the original exception as the "previous" exception.

I'm already working on a fix :)

**Comment by mpociot** [comment]
This is now fixed in 1.4.15 👍

**Comment by jasonvarga** [comment]
Just like that. Champion. 🥇 

Thanks!
